src/main/java/pl/nogacz/chess/application/Design.java

```diff
@@ -36,7 +40,16 @@ public Design() {
         borderPane.setRight(vBox);
         borderPane.setTop(hBox);
     }
-
+    
+    public static void setWinChance(int calc){
+        calc = (calc+100) / 2;
+        if(calc > 100)
+            calc = 100;
+        if(calc < 0)
+            calc = 0;
+        winChance.setValue(calc);

```

Comment:
Bu farkı doğrudan bu şekilde oranlamadan kullanmak mantıklı mı? calculateBoardPoint ne gibi değerler dönüyor? Genelde 0 ile 100 arası mı?

Reply:
calculateBoardPoint metodu hemen hemen her durumda -100 ile +100 arası değerler dönüyor, ben bunu slider limitleri(0 - 100) arasında göstermek için  calc = (calc+100) / 2; operasyonunu gerçekleştiriyorum. (calc = calculateBoardPoint return value)
     Onlarca kez yaptığım denemelerde bu değerin 0 ile 100 dışına ancak şah çekildiği zaman taştığını gördüm. Bu da her şah çekildiğinde yenme şansı göstergesinin gerçekçi olmayan değerlere inmesine veya çıkmasına sebep oluyor.  Bu yüzden bu durumları engelledim.